mkdocs.yml

```diff
@@ -1,50 +1,51 @@
 site_name: DRESSCode
 site_description: Data Reduction of Extended SWIFT Sources Code
+site_url: https://spacetelescope.github.io/DRESSCode/
```

Comment:
this is a (newly?) required param for mkdocs. the rest of the edits in this file are just whitespace changes, made by my autoformatter.